ozaria/site/api/tint.js

```diff
@@ -0,0 +1,20 @@
+import fetchJson from 'app/core/api/fetch-json'
```

Comment:
Only exposing the `PUT` interface as there isn't a need to support the creation or `POST` interface. The two initial tint groups can be made manually.